base/applications/cacls/lang/id-ID.rc

```diff
@@ -41,32 +41,32 @@ Singkatan:\n\
     IDS_ABBR_CHANGE "C"
     IDS_ABBR_NONE "N"
     IDS_ALLOW ""
-    IDS_DENY "(DENY)"
+    IDS_DENY "(MENOLAK)"
     IDS_SPECIAL_ACCESS "(akses spesial:)"
-    IDS_GENERIC_READ "GENERIC_READ"
-    IDS_GENERIC_WRITE "GENERIC_WRITE"
-    IDS_GENERIC_EXECUTE "GENERIC_EXECUTE"
-    IDS_GENERIC_ALL "GENERIC_ALL"
-    IDS_FILE_GENERIC_EXECUTE "FILE_GENERIC_EXECUTE"
-    IDS_FILE_GENERIC_READ "FILE_GENERIC_READ"
-    IDS_FILE_GENERIC_WRITE "FILE_GENERIC_WRITE"
-    IDS_FILE_READ_DATA "FILE_READ_DATA"
-    IDS_FILE_WRITE_DATA "FILE_WRITE_DATA"
-    IDS_FILE_APPEND_DATA "FILE_APPEND_DATA"
-    IDS_FILE_READ_EA "FILE_READ_EA"
-    IDS_FILE_WRITE_EA "FILE_WRITE_EA"
-    IDS_FILE_EXECUTE "FILE_EXECUTE"
-    IDS_FILE_DELETE_CHILD "FILE_DELETE_CHILD"
-    IDS_FILE_READ_ATTRIBUTES "FILE_READ_ATTRIBUTES"
-    IDS_FILE_WRITE_ATTRIBUTES "FILE_WRITE_ATTRIBUTES"
-    IDS_MAXIMUM_ALLOWED "MAXIMUM_ALLOWED"
-    IDS_ACCESS_SYSTEM_SECURITY "ACCESS_SYSTEM_SECURITY"
-    IDS_SPECIFIC_RIGHTS_ALL "SPECIFIC_RIGHTS_ALL"
-    IDS_STANDARD_RIGHTS_REQUIRED "STANDARD_RIGHTS_REQUIRED"
+    IDS_GENERIC_READ "BACAAN-UMUM"
+    IDS_GENERIC_WRITE "TULISAN-UMUM"
+    IDS_GENERIC_EXECUTE "JALANKAN-UMUM"
+    IDS_GENERIC_ALL "SEMUA-UMUM"
+    IDS_FILE_GENERIC_EXECUTE "JALANKAN-BERKAS-UMUM"
+    IDS_FILE_GENERIC_READ "BERKAS-BACAA-UMUM"
+    IDS_FILE_GENERIC_WRITE "BERKAS-TULISAN-UMUM"
+    IDS_FILE_READ_DATA "BERKAS-BACAAN-DATA"
+    IDS_FILE_WRITE_DATA "BERKAS-TULISAN-DATA"
+    IDS_FILE_APPEND_DATA "BERKAS-MENAMBAHKAN-DATA"
+    IDS_FILE_READ_EA "BERKAS-BACAAN-EA"
+    IDS_FILE_WRITE_EA "BERKAS-MENULIS-DATA"
+    IDS_FILE_EXECUTE "BERKAS-MENJALANKAN"
+    IDS_FILE_DELETE_CHILD "BERKAS-MENGHAPUS-ANAK"
+    IDS_FILE_READ_ATTRIBUTES "BERKAS-BACAAN-ATRIBUT"
+    IDS_FILE_WRITE_ATTRIBUTES "BERKAS-TULISAN-ATRIBUT"
+    IDS_MAXIMUM_ALLOWED "MAKSIMUM-DIZINKAN"
+    IDS_ACCESS_SYSTEM_SECURITY "MENGAKSES-SISTEM-KEAMANAN"
+    IDS_SPECIFIC_RIGHTS_ALL "SPESIFIK-HAK-SEMUA"
+    IDS_STANDARD_RIGHTS_REQUIRED "STANDAR-HAK-WAJIB"
     IDS_SYNCHRONIZE "SYNCHRONIZE"
-    IDS_WRITE_OWNER "WRITE_OWNER"
-    IDS_WRITE_DAC "WRITE_DAC"
-    IDS_READ_CONTROL "READ_CONTROL"
-    IDS_DELETE "DELETE"
-    IDS_STANDARD_RIGHTS_ALL "STANDARD_RIGHTS_ALL"
+    IDS_WRITE_OWNER "MENULIS-OWNER"
+    IDS_WRITE_DAC "MENULIS-DAC"
+    IDS_READ_CONTROL "MEMBACA-KONTROL"
+    IDS_DELETE "MENGHAPUS"
+    IDS_STANDARD_RIGHTS_ALL "STANDAR-HAK-SEMUA"
```

Comment:
My remark stops here.

Reply:
Then why are they in resource files? please review the rest hbelusca